### DISCUSSION TITLE
Import Solution Failed: Object reference not set to an instance of an object

### DISCUSSION THREAD

**Discussion by UncleBoost** [microsoft/powercat-creator-kit#220] [Q&A] [answered]
I get this error when importing the latest version of the creator kit. Any ideas of how to fix it?
I can install earlier versions OK, though the other solution I need to import has a dependency on the latest version of the creator kit.
Please help!

**Comment by Zakchap2020** [comment]
I am having the same issue

**Comment by fishyconnor** [comment]
Same issue, trying to import CreatorKit Nov 2022, as part of the upgrade from CoE Nov 2022 to Dec 2022 release as per release notes.

**Comment by denise-msft** [comment, marked as answer]
Thanks for reporting this. Our dev team tested this week and reported no issues.

Please try importing again and let us know if you're still having trouble importing. If you do still get an import error, please share your region so we can investigate.
